airflow-core/src/airflow/ui/src/pages/DagsList/DagCard.test.tsx

```diff
@@ -83,7 +83,7 @@ describe("DagCard", () => {
     render(<DagCard dag={expandedMockDag} />, { wrapper: Wrapper });
     expect(screen.getByTestId("dag-tag")).toBeInTheDocument();
     expect(screen.queryByText("tag4")).toBeInTheDocument();
```

Comment:
Maybe we can adjust the test to ensure that the last 2 are not concatenated. For instance check for `tag3, tag4` in the document. And not just `tag4`